dev-docs/operations/operations/zuul-ci-cd-quickstart-user-guide.md

```diff
@@ -307,6 +309,9 @@ The content may look like this:
 You may use this content or the file to provide it as a secret. You just have to update the `<name>` and the
 `<fieldname>` part.
 
+**NOTE!** The secret name has to be unique across all projects, so if you are unsure, it's
+best to disambiguate via a prefix or suffix that is related to the name of your repository.
```

Comment:
I would prefer that we introduce a prefix `SECRET_REPO` for all secrets in a specific repository as a best practice.

Reply:
So, in the standards repo, the prefix would be (for instance) `SECRET_STANDARDS_application_credentials`? Who can decide such a matter? The OPS team? I can put it on the agenda.

Reply:
thanks @mbuechse for adding it to the agenda - very helpful!

Reply:
Has been discussed in todays OPS call. @berendt's proposal will be documented. #461.

Reply:
Added my proposal to this PR with https://github.com/SovereignCloudStack/docs/pull/96/commits/dfe81e7017361c29dfa063c52aae4760b0da6166. @mbuechse Please have a look on it.

Reply:
Fine by me. I would add one sentence, see my "review".